contracts/test/strategies/compoundingSSVStaking.js

```diff
@@ -348,12 +348,23 @@ describe("Unit test: Compounding SSV Staking Strategy", function () {
         },
         depositGwei
       );
+
+    if (state === "STAKED") return stakeTx;
+
+    const verifiedDepositTx = await verifyDeposit(testValidator);
+
+    if (state === "VERIFIED_DEPOSIT") return verifiedDepositTx;
+
+    throw Error(`Invalid state: ${state}`);
+  };
+
+  const verifyDeposit = async (testValidator) => {
+    const { beaconRoots, compoundingStakingSSVStrategy } = fixture;
+
     const { pendingDepositRoot, depositSlot } = await getLastDeposit(
       compoundingStakingSSVStrategy
     );
 
-    if (state === "STAKED") return stakeTx;
```

Comment:
Why does this need removing? There are number of tests that put the deposit in stakes but not verified state